src/test/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitCheckerTest.java

```diff
@@ -260,7 +260,7 @@ public void ThrottleForNormalizeTestWithinIdeal() throws Exception {
         assertEquals(1, countOfOutputLinesContaining("under budget"));
 
         // The last scenario will trigger back to under budget with a full limit but no new messages
-        assertEquals(8, handler.getView().size());
+        assertEquals(10, handler.getView().size());
```

Comment:
@car-roll  This number changed, I'm guess because of the added "Jenkins is using..." messages, but it would be nice to have an assert after line 260 to show that.